app/scripts/HorizontalLine1DPixiTrack.js

```diff
@@ -167,7 +167,11 @@ class HorizontalLine1DPixiTrack extends HorizontalTiled1DPixiTrack {
 
     for (let i = 0; i < tileValues.length; i++) {
       const xPos = this._xScale(tileXScale(i));
-      const yPos = this.valueScale(tileValues[i] + offsetValue);
+      let yPos = this.valueScale(tileValues[i] + offsetValue);
+
+      if (Number.isNaN(tileValues[i]) && this.options.nanAsZero) {
```

Comment:
How about combining this to

```js
const yValue = this.options.nanAsZero && Number.isNaN(tileValues[i]) ? 0 : tileValues[i];
const yPos = this.valueScale(yValue + offsetValue);
```

While functionally equivalent, I personally like more as it signals to the developer: yValue and yPos are two fixed variables that cannot be changed later on. While with `let` the value could be changed elsewhere and so one would have to example the entire file.

Reply:
I like it! Thanks for the suggestion. Just included it and pushed a change.